parquet-column/pom.xml

```diff
@@ -77,6 +77,11 @@
       <version>1.3.149</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
why does this need to be added? isn't this inherited from the parent pom?
Shouldn't this have `<scope>test</scope>` ? Or is this being added to add it as a requirement at runtime (not only for tests)?